src/github/githubRepository.ts

```diff
@@ -345,6 +345,25 @@ export class GitHubRepository implements vscode.Disposable {
 		}
 	}
 
+	async deleteBranch(pullRequestModel: PullRequestModel): Promise<void> {
+		const { octokit } = await this.ensure();
+
+		if (!pullRequestModel.isResolved()) {
+			return;

```

Comment:
I think it would also make sense to show a notification in this case

Reply:
yup, `validatePullRequestModel` will show notification.